src/framework/mlt_frame.c

```diff
@@ -445,6 +470,9 @@ int mlt_image_format_size( mlt_image_format format, int width, int height, int *
 		case mlt_image_glsl_texture:
 			if ( bpp ) *bpp = 0;
 			return 4;
+		case mlt_image_yuv422p16:
+			if ( bpp ) *bpp = 2;
```

Comment:
By looking at the other formats, I assume that bpp is meant to indicate the overall bytes per pixel. That is, bpp = total bytes in the entire image / total pixels in the entire image.